frontend/test/metabase-visual/dashboard/parameters-widget.cy.spec.js

```diff
@@ -0,0 +1,62 @@
+import { restore } from "__support__/e2e/cypress";
+import { SAMPLE_DATASET } from "__support__/e2e/cypress_sample_dataset";
+
+const { PRODUCTS, PRODUCTS_ID } = SAMPLE_DATASET;
+
+const questionDetails = {
+  query: { "source-table": PRODUCTS_ID },
+};
+
+const filter = {
+  name: "Category",
+  slug: "category",
+  id: "ad1c877e",
+  type: "category",
+};
+
+const filters = new Array(12).fill(filter);
+
+describe("visual tests > dashboard > parameters widget", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+
+    cy.createQuestionAndDashboard({ questionDetails }).then(
+      ({ body: { id, card_id, dashboard_id } }) => {
+        cy.request("PUT", `/api/dashboard/${dashboard_id}`, {
+          parameters: filters,
+        });
+
+        // Connect filter to the card
+        cy.request("PUT", `/api/dashboard/${dashboard_id}/cards`, {
+          cards: [
+            {
+              id,
+              card_id,
+              row: 0,
+              col: 0,
+              sizeX: 12,
+              sizeY: 32,
+              visualization_settings: {},
+              parameter_mappings: [
+                {
+                  parameter_id: "ad1c877e",
+                  card_id,
+                  target: ["dimension", ["field-id", PRODUCTS.CATEGORY]],
+                },
+              ],
+            },
+          ],
+        });
+
+        cy.visit(`/dashboard/${dashboard_id}`);
+      },
+    );
```

Comment:
Heads-up:
Now that the https://github.com/metabase/metabase/pull/17493 is merged, you can simplify this part greatly.

We don't care about connecting that filter to the card, right? So we can omit `parameter_mappings` so the only thing left to do is to resize the card:

```javascript
cy.createQuestionAndDashboard({ questionDetails }).then(
  ({ body: oldCard }) => {
    const { dashboard_id } = oldCard;

    cy.request("PUT", `/api/dashboard/${dashboard_id}`, {
      parameters: filters,
    });

    const updatedSize = {
      sizeX: 12,
      sizeY: 32,
    };

    cy.editDashboardCard(oldCard, updatedSize);

    cy.visit(`/dashboard/${dashboard_id}`);
  },
);
```

It's absolutely not necessary to update this test. Just wanted to give you an example in case it looks interesting :)